README.md

```diff
@@ -1,19 +1,18 @@
 # ZeroTier-GUI
+<img src="zerotier-gui.png">
 
+[![GitHub release (latest SemVer)](https://img.shields.io/github/v/release/tralph3/ZeroTier-GUI)](https://github.com/tralph3/ZeroTier-GUI/releases)
 [![License: GPL v3](https://img.shields.io/badge/License-GPL%20v3-blue.svg)](https://github.com/tralph3/ZeroTier-GUI/blob/master/LICENSE)
+[![Send me a small gift](https://img.shields.io/badge/send%20me%20a%20small%20gift-paypal-blue.svg)](https://paypal.me/tralph3)
```

Comment:
Neat idea about the badges! However I saw that there's a flat version of these that I like better aesthetically. I'd say replacing these with those is better.

Reply:
Just adding `?style=flat-square` to the end of the image sources is enough.